modules/apps/content-dashboard/content-dashboard-document-library-impl/src/test/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItemTest.java

```diff
@@ -375,6 +518,26 @@ public void testIsViewable() throws Exception {
 				_getHttpServletRequest(RandomTestUtil.randomLong())));
 	}
 
+	private void _assertContentDashboardItemVersionAction(
+		ContentDashboardItemVersionAction
```

Comment:
Hey @jkappler !!

Can we use the equals method in ContentDashboardItemVersionAction for this?

Thanks!!

Reply:
Will change this @cgoncas!